atomic_scanners/misc-package-updates/scanner.py

```diff
@@ -185,7 +185,7 @@ def template_json_data(scan_type):
         json_out["Scan Results"] = \
             "Could not find {} executable in the image".format(cli_arg)
         json_out["Successful"] = "false"
-        json_out["Summary"] = "No updates for packages installed via {}".\
+        json_out["Summary"] = "No updates for packages installed via {}. ".\
```

Comment:
how's this file part of this commit (message)?

Reply:
I don't understand the question here.

Reply:
Check the commit message and the file in question, I think the commit message is not destined for the file updated.

Reply:
Apologies. Will be careful henceforth.